### PR TITLE
Nuget

### PR DESCRIPTION
**Pull request by frenzqse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-Project containing all files common for all loom-csharp implementations.
+Requirements
+=================
+Visual studio 2012
+or
+Visual studio and nuget
+
+Summary
+=================
+While opening the Bridge.sln, Visual studio automatically restores the nuget packages.
```
Comment (anpieber) [inline README.md]: A little bit more readme would be nice :-)

**Comment by anpieber** [comment]
Basically +1 to all of it; the only thing I'm not quite sure of is the versioning. how do keep the versions of those projects and assemblies in sync? Or is this all a "visual studio thing"?

File: README.md (hunk 1 above)
Comment (frenzqse) [inline README.md]: But there ist not more to explain :-)

Comment (anpieber) [inline README.md]: At least a link to the documentation? What to open? A little bit more? :-)

**Comment by anpieber** [comment]
pls dont forget my "general" comment :-)

**Comment by anpieber** [comment]
a) Maybe we can keep the readme file more general like here (https://github.com/openengsb-labs/labs-liquibase)
b) What about my versioning question 3 days ago?
c) Which wrapped projects could be removed once this pull request had been merged?

**Comment by frenzqse** [comment]
a) The thing is, for the bridge is really not more. Nuget is doeing everything and executing the test can be done in visual studio or it is documentated in the documentation. There is no build or anything else. The readme of th wsdl2dll will be much more detailed.
b) I responsed it (I think so :-)). For the domains, the versioning is done over the plugin an for the bridge itselfe, the assembly.cs defines the version.
c)
https://github.com/openengsb/openengsb-wrapped-csharp-Apache.NMS.ActiveMQ
https://github.com/openengsb/openengsb-wrapped-csharp-Apache.NMS.API
https://github.com/openengsb/openengsb-wrapped-csharp-Apache.log4net
https://github.com/openengsb/openengsb-wrapped-csharp-codeplex.fastjson
https://github.com/openengsb/openengsb-wrapped-csharp-codeplex.newton.json

**Comment by anpieber** [comment]
@a) Fair enough, but still exactly those words would help: "To build and test use visual studio; since the project is based on msbuild and mstest" :-)
@b) Yeah, I got that; but there is more than one assembly.cs and how do you keep those in sync? Is there something in visual studio I simply don't know?
@c) great thanks; I'll remove those once we merge this pull request

**Comment by frenzqse** [comment]
a) ok will do it directly
b) Good question. I think that there is no automatique way to keep them sync. I think we have to do that by hand.

**Comment by anpieber** [comment]
at readme: GREAT! That was exactly what I was looking for :-)
at assembly.cs: damn you c#/visual studio/microsoft -.- OK, well... then let's hope we don't make a mistake in this...

I'll merge this pull request later this day and also remove the no longer required repositories! Thank you very much

**Comment by anpieber** [comment]
I start to move the no longer required projects

**Pull request merged**